src/components/CardContainer.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 import Card from './Card'
 // import MovieDetails from './MovieDetails'
-import './CardContainer.css'
+import '../styles/CardContainer.css'
 
 const CardContainer = (props) => {
```

Comment:
Try destructuring your props so you don't have to keep using `props.` when you reference them